modules/aerodyn/src/AeroDyn.f90

```diff
@@ -2577,6 +2595,11 @@ subroutine CalcBuoyantLoads( u, p, m, y, ErrStat, ErrMsg )
    if ( p%VolNac == 0 ) then
       m%NacFB = NacFBtmp
       m%NacMB = NacMBtmp
+
+      ! Passing buoyant loads to m variable, drag loads are called after buoyant loads
+      m%NacFi = NacFBtmp
```

Comment:
Since NacFi and NacMi represent the total load at the nacelle, would it be better to set them equal to y%NacelleLoad after line 2646, similar to what is done in the computeNacelleDrag subroutine?

Reply:
Nice catch! implemented similar to `computeNacelleDrag`